### ISSUE TITLE
[Bug]: Crash on opening *.kts files

### ISSUE THREAD

**Issue by Rafael2616** [AndroidIDEOfficial/AndroidIDE#1016]
### What happened?

When I go to open a Kotlin file, it's usually when it happens the first files of the day, that I try to open. This error started to occur after updating sora editor to the latest snapshot a few commits ago, but I'm not entirely sure about that

### What's the expected behavior?

don't crash

### What version of AndroidIDE you're using?

v2.4.0 (debug builds)

### Relevant log output

```shell
AndroidIDE crash report
Manufacturer: LGE
Device: LM-X420
ABI: armeabi-v7a
SDK version: 28
App version: v2.4.1-beta.internal.1+branch.dev.commit.799d6f7.timestamp.20230601172949 (241)

 Stacktrace: 
java.lang.IllegalArgumentException: query(name:brackets) parsing failed: Structure at text offset 3
	at io.github.rosemoe.sora.editor.ts.UtilsKt.validateOrThrow(Utils.kt:39)
	at io.github.rosemoe.sora.editor.ts.TsLanguageSpec.<init>(TsLanguageSpec.kt:118)
	at com.itsaky.androidide.editor.schemes.LanguageSpecProvider.getLanguageSpec(LanguageSpecProvider.kt:51)
	at com.itsaky.androidide.editor.language.treesitter.TreeSitterLanguage.<init>(TreeSitterLanguage.kt:58)
	at com.itsaky.androidide.editor.language.kotlin.KotlinLanguage.<init>(KotlinLanguage.kt:30)
	at com.itsaky.androidide.editor.language.kotlin.KotlinLanguage.FACTORY$lambda$0(KotlinLanguage.kt:33)
	at com.itsaky.androidide.editor.language.kotlin.KotlinLanguage.$r8$lambda$AzoEE-HedI8ey8DrFqnPQ2fZJYM(Unknown Source:0)
	at com.itsaky.androidide.editor.language.kotlin.KotlinLanguage$$ExternalSyntheticLambda0.create(Unknown Source:0)
	at com.itsaky.androidide.editor.language.treesitter.TreeSitterLanguageProvider.forFile(TreeSitterLanguageProvider.kt:34)
	at com.itsaky.androidide.editor.ui.IDEEditor.createLanguage(IDEEditor.java:985)
	at com.itsaky.androidide.editor.ui.IDEEditor.setupLanguage(IDEEditor.java:947)
	at com.itsaky.androidide.ui.editor.CodeEditorView.postRead(CodeEditorView.java:163)
	at com.itsaky.androidide.ui.editor.CodeEditorView.lambda$new$0$com-itsaky-androidide-ui-editor-CodeEditorView(CodeEditorView.java:131)
	at com.itsaky.androidide.ui.editor.CodeEditorView$$ExternalSyntheticLambda0.run(Unknown Source:8)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:205)
	at android.app.ActivityThread.main(ActivityThread.java:6993)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:884)
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by itsaky** [comment]
Fixed in 876903c0e0f3d705e912765b3e531eab5602076e.

**Comment by Rafael2616** [comment]
it worked, I tried to reproduce the bug again several times and it doesn't happen anymore👍
